lib/ansible/modules/files/replace.py

```diff
@@ -224,7 +224,11 @@ def main():
         module.fail_json(rc=257, msg='Path %s does not exist !' % path)
     else:
         f = open(path, 'rb')
-        contents = to_text(f.read(), errors='surrogate_or_strict')
+        data = f.read()
+        try:
```

Comment:
You can use `to_native`. See [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/_text.py#L273)